lib/highline/import.rb

```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/env ruby
```

Comment:
@abinoam hey, issued a PR pointing to your repository https://github.com/abinoam/highline/pull/2
So if that gets merged it should automatically appear here.